src/common/commandExecutor.ts

```diff
@@ -36,8 +38,7 @@ export enum CommandStatus {
 
 export class CommandExecutor {
 
-    private static ReactNativeCommand = "react-native";
-    private static ReactNativeVersionCommand = "--version";
+    public static ReactNativeCommand: string | null;
```

Comment:
This is defined here but initialized some other place? Why is that?

Reply:
`ReactNativeCommand` variable is responsible for React Native global CLI command name. If `ReactNativeCommand` isn't empty we use global CLI for commands execution, otherwise local CLI approach is used. React Native global CLI command name is set in `settings.json` file. `settings.json` file is parsed in `ExtensionServer` in `requestSetup` function and also in `CommandPaletteHandler` in `getRunOptions` function. Therefore we consider that it can be appropriate and conveniently to assign `ReactNativeCommand` parameter in these methods. We can't get this parameter directly from `CommandExecutor`.